packages/yourloops/components/language-select.tsx

```diff
@@ -34,54 +34,55 @@ import { Theme } from '@mui/material/styles'
 import { makeStyles } from 'tss-react/mui'
 import { useTranslation } from 'react-i18next'
 
-const languageSelectStyle = makeStyles()((theme: Theme) => {
-  return {
-    select: {
-      fontSize: '12px',
-      color: theme.palette.grey[700]
-    }
+interface LanguageSelectProps {
+  className?: string
+}
+
+const styles = makeStyles()((theme: Theme) => ({
+  select: {
+    fontSize: '12px',
+    color: theme.palette.grey[700],
+    paddingBlock: 0
   }
-})
+}))
 
-function LanguageSelect(): JSX.Element {
+const LanguageSelect: FunctionComponent<LanguageSelectProps> = ({ className }) => {
   const { i18n } = useTranslation()
-  const [val, setVal] = React.useState(i18n.language)
-  const { classes } = languageSelectStyle()
+  const [val, setVal] = useState(i18n.language)
+  const { classes, cx } = styles()
+  const languages = Object.entries(i18n.options.resources)
```

Comment:
I have a suggestion to have a better object model to handle here (I had a hard time understanding the current one):

```ts
const languages = Object.entries(i18n.options.resources).map((languageEntry: [string, ResourceLanguage]) => {
  return { key: languageEntry[0], name: languageEntry[1].name }
})
```

And then in the HTML:
```tsx
languages.map(language => (
  <MenuItem
    id={`language-selector-${language.key}`}
    key={language.key}
    value={language.key}
  >
    {language.name as string}
  </MenuItem>
))}
```

What do you think?

Reply:
👍🏻